MyOnboarding/src/MyOnboardingApp.Validation/Validators/Validator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using MyOnboardingApp.Contracts.Validation;
+
+namespace MyOnboardingApp.Validation.Validators
+{
+    internal class Validator<T> : IInvariantValidator<T>
```

Comment:
I would prefer the "invariant" word to participate in the class name here:
```suggestion
    internal class InvariantValidator<T> : IInvariantValidator<T>
```